Dockerfile

```diff
@@ -2,37 +2,113 @@
 # This Dockerfile requires BuildKit to be enabled, by setting the environment variable
 # DOCKER_BUILDKIT=1
 # see https://docs.docker.com/build/buildkit/#getting-started
-ARG JVM="eclipse-temurin"
-ARG JAVA_VERSION="17"
-# We duplicate the JVM and JAVA_VERSION vars here as renovate will otherwise fail to properly parse
-ARG BASE_IMAGE="eclipse-temurin:17-jre-focal"
-ARG BASE_DIGEST_AMD64="sha256:e3a9caf145a274335a402ea48f37a88450a847bec6effbd98f61c0c552ee9b53"
-ARG BASE_DIGEST_ARM64="sha256:737781ca3eb1bc00e7b322a53f45f406a28fd57b030ee58760ef0c3495fb9c6e"
+ARG BASE_IMAGE="ubuntu:jammy-20230624"
```

Comment:
I wonder if renovate supports this properly? Can check it's logs after.

Reply:
I wish we could somehow test renovate locally or something :(